src/index.js

```diff
@@ -1,10 +1,25 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+import { render } from 'react-dom';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import './index.css';
 import App from './App';
+import { Nav } from './components/Nav';
+import { AddItems } from './pages/Add-items/Add-items';
+import { List } from './pages/List/List';
 import * as serviceWorker from './serviceWorker';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+const rootElement = document.getElementById('root');
+render(
+  <BrowserRouter>
```

Comment:
just curious if there is any specific reason to why you guys placed the `Router` in `index.js` instead of `app.js`?

Reply:
Ohhh thanks, Camila, we searched some documentation about it, but I would keep improving this issue with your feedback